client/src/commponet/Description.js

```diff
@@ -0,0 +1,93 @@
+import { useState } from "react";
+// import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
Generally you don't want to include commented out code in your commits unless there's a good reason to.